.github/workflows/ci.yml

```diff
@@ -379,7 +378,10 @@ jobs:
                   else
                     echo "No report will be created for '$GITHUB_EVENT_NAME' event"
                   fi
-                
+
+            - name: 'Add annotation to workflow run linking to PR'
```

Comment:
I think it would work better in the `e2e-test-reports` job.

Reply:
Maybe it's just Monday, but isn't this in the `e2e-test-reports` job?  It's the last step before the `api-test-reports` job.

Reply:
You are right, I was confused by the the code being collapsed in a very unfortunate way in the review UI. It was masking `e2e-test-reports` and showing `alert-on-failure` right above... 🤦🏼 
Sorry about that.